src/components/Votes.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from "react"
+import { getArticleById, upVoteArticle, downVoteArticle } from "../api"
+
+const Votes = ({article_id}) => {
+    const [votes, setVotes] = useState(0)
+    const [isLiked, setIsLiked] = useState(false)
+    const [isDisliked, setIsDisliked] = useState(false)
+    
+useEffect(() => {
+    getArticleById(article_id).then(({article}) => {
```

Comment:
we already have access to the votes in this component's parent component - could we just pass this down rather than make a whole new api request for it?